website/src/pages/dashboard.tsx

```diff
@@ -1,34 +1,24 @@
 import { Flex } from "@chakra-ui/react";
 import Head from "next/head";
 import { useTranslation } from "next-i18next";
-import { useEffect, useMemo, useState } from "react";
+import { useMemo } from "react";
 import { LeaderboardWidget, TaskOption, WelcomeCard } from "src/components/Dashboard";
 import { getDashboardLayout } from "src/components/Layout";
 import { get } from "src/lib/api";
 import { AvailableTasks, TaskCategory } from "src/types/Task";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
 import { TaskCategoryItem } from "src/components/Dashboard/TaskOption";
+import { useCurrentLocale } from "src/hooks/locale/useCurrentLocale";
+import { API_ROUTES } from "src/lib/routes";
 import useSWR from "swr";
 
 const Dashboard = () => {
-  const {
-    t,
-    i18n: { language },
-  } = useTranslation(["dashboard", "common", "tasks"]);
-  const [activeLang, setLang] = useState<string>(null);
-  const { data, mutate: fetchTasks } = useSWR<AvailableTasks>("/api/available_tasks", get, {
+  const { t } = useTranslation(["dashboard", "common", "tasks"]);
+  const lang = useCurrentLocale();
+  const { data } = useSWR<AvailableTasks>(API_ROUTES.AVAILABLE_TASK({ lang }), get, {
     refreshInterval: 2 * 60 * 1000, //2 minutes
-    revalidateOnMount: false, // triggered in the hook below
   });
 
-  useEffect(() => {
-    // re-fetch tasks if the language has changed
-    if (activeLang !== language) {
-      setLang(language);
-      fetchTasks();
-    }
-  }, [activeLang, setLang, language, fetchTasks]);
-
```

Comment:
I have seen that you updated the `useSWR`, will it auto fetch?

Otherwise the cards on the dashboard will be wrong.

Reply:
Yes, revalidateOnMount: true which is global config will fetch it by default

Reply:
great